packages/protocol/migrations_sol/Migration.s.sol

```diff
@@ -296,50 +310,49 @@ contract Migration is Script, UsingRegistry, MigrationsConstants {
   }
 
   function migrateFeeCurrencyDirectory() public {
-    deployProxiedContract(
+    address feeCurrencyDirectoryProxyAddress = 0x9212Fb72ae65367A7c887eC4Ad9bE310BAC611BF;
+    deployImplementationAndAddToRegistry(
       "FeeCurrencyDirectory",
+      IProxy(feeCurrencyDirectoryProxyAddress),
       abi.encodeWithSelector(IFeeCurrencyDirectoryInitializer.initialize.selector)
     );
+
+    addToRegistry("FeeCurrencyDirectory", feeCurrencyDirectoryProxyAddress);
   }
 
   function migrateCeloToken(string memory json) public {
-    // TODO change pre-funded addresses to make it match circulation supply
-    address celoProxyAddress = deployProxiedContract(
+    // TODO: change pre-funded addresses to make it match circulation supply
+    // pre deployed celo token proxy address from L2Genesis.s.sol
+    address celoProxyAddress = 0x471EcE3750Da237f93B8E339c536989b8978a438;
```

Comment:
same here as for FeeCurrencyDirectory (only that we hardcode it in our migrationConfig)